priv/repo/migrations/20210205030432_create_submission_votes_table.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Cadet.Repo.Migrations.AddSubmissionVotesTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:submission_votes) do
+      add(:score, :integer)
+      add(:user_id, references(:users), null: false)
+      add(:submission_id, references(:submissions), null: false)
+      add(:assessment_id, references(:assessments), null: false)
```

Comment:
Assessment ID is redundant since each submission belongs to an assessment. Please remove the column.